doc/specs/thrift-compact-protocol.md

```diff
@@ -249,7 +249,7 @@ The short form should be used when the length is in the range 0 - 14 (inclusive)
 
 The following element-types are used (see note below):
 
-* `BOOL`, encoded as `2`
+* `BOOL`, encoded as `1`
```

Comment:
That is debatable. A better word could be inconcistent.

* thrift_compact_protocol.erl does it acciotrding to the spec and writes 2
* anyone else writes 1